src/lib/variants.ts

```diff
@@ -195,6 +215,7 @@ export const findVariant = (data: EpicTests, targeting: EpicTargeting): Result |
         excludeSection,
         excludeTags,
         hasCountryCode,
+        matchesCountryGroups,
```

Comment:
What's the interaction between this and `hasCountryCode`? It looks like maybe there's some overlap?

Reply:
They're different filters and serve different purposes, which I agree is a bit confusing but it's for good reason and consistent with Frontend. `hasCountryCode` is to make sure we know the user's country if the test template uses the country name placeholder. There's a property in the test object used for this specifically.
`matchesCountryGroups` is specifically for geotargeting of the test itself.
But looking through this I did realise an improvement that could be done on the `hasCountryCode` test to make sure the logic is exactly the same as in Frontend, so I made a tweak on that.